lib/logstash/outputs/http.rb

```diff
@@ -89,31 +89,29 @@ def register
     validate_format!
   end # def register
 
-  # TODO: (colin) the request call sequence + async handling will have to be reworked when using
-  # Maticore >= 5.0. I will set a version constrain in the gemspec for this.
-  def receive(event)
+  def multi_receive(events)
+    events.each {|event| receive(event, :parallel)}
+    client.execute!
+  end
+
+  # Once we no longer need to support Logstash < 2.2 (pre-ng-pipeline)
+  # We don't need to handle :background style requests
```

Comment:
Well, that's a manticore-ism. In manticore client.background returns a backgrounding proxy. It's not an HTTP proxy, but a proxy object. In this context I should rename it to `request_async_type` for clarity.